website/content/docs/nia/configuration.mdx

```diff
@@ -127,8 +130,20 @@ task {
-- `variable_files` - (list[string]) A list of paths to files containing variables for the task. For the [Terraform driver](#terraform-driver), these are used as Terraform [variable defintion (`.tfvars`) files](https://www.terraform.io/docs/configuration/variables.html#variable-definitions-tfvars-files) and consists of only variable name assignments. The variable assignments must match the corresponding variable declarations available by the Terraform module for the task. Consul-Terraform-Sync will generate the intermediate variable declarations to pass as arguments from the auto-generated root module to the task's module. Variables are loaded in the same order as they appear in the order of the files. Duplicate variables are overwritten with the later value. _Note: unless specified by the module, configure arguments for Terraform providers using [`terraform_provider` blocks](#terraform-provider)._
+- `source` - (string: required) Source is the location the driver uses to discover the Terraform module used for automation. The source is the module path which can be local or remote on the [Terraform Registry](https://registry.terraform.io/) or private module registry. Read more on [Terraform module source and other supported types here](https://www.terraform.io/docs/modules/sources.html).
+  - To use a private module with the [`terraform` driver](#terraform-driver), run the command [`terraform login [hostname]`](https://learn.hashicorp.com/tutorials/terraform/cloud-login) to authenticate the local Terraform CLI prior to starting Consul-Terraform-Sync.
+  - To use a private module with the [`terraform_cloud` driver](#terraform-cloud-driver), no extra steps are needed.
+    ```hcl
+    // local module example: "./terraform-cts-hello"
+    source = "<PATH>"
+
+    // public module example: "mkam/hello/cts"
+    source = "<NAMESPACE>/<MODULE NAME>/<PROVIDER>"
+
+    // private module example: "my.tfe.hostname.io/my-org/hello/cts"
+    source = "<HOSTNAME>/<ORGANIZATION>/<MODULE NAME>/<PROVIDER>"
+    ```
+- `variable_files` - (list[string]) A list of paths to files containing variables for the task. For the [Terraform driver](#terraform-driver), these are used as Terraform [variable definition (`.tfvars`) files](https://www.terraform.io/docs/configuration/variables.html#variable-definitions-tfvars-files) and consists of only variable name assignments. The variable assignments must match the corresponding variable declarations available by the Terraform module for the task. Consul-Terraform-Sync will generate the intermediate variable declarations to pass as arguments from the auto-generated root module to the task's module. Variables are loaded in the same order as they appear in the order of the files. Duplicate variables are overwritten with the later value. _Note: unless specified by the module, configure arguments for Terraform providers using [`terraform_provider` blocks](#terraform-provider)._
```

Comment:
The intention is the files themselves that contains Terraform values.

The abstraction here is residue from us originally writing the docs with flexibility not pinned to Terraform verbiage but that has since changed. We are ok with having Terraform specific verbiage threaded in our docs.

Thanks for pointing out how unclear this portion is for us to revisit! So here's another jab at this section

```suggestion
- `variable_files` - (list[string]) Specifies list of paths to [Terraform variable definition files (`.tfvars`)](https://www.terraform.io/docs/configuration/variables.html#variable-definitions-tfvars-files). The content of these files should consist of only variable name assignments. The variable assignments must match the corresponding variable declarations made available by the Terraform module for the task. 
  - Variables are loaded in the order they appear in the files. Duplicate variables are overwritten with the later value. _Unless specified by the module, configure arguments for Terraform providers using [`terraform_provider` blocks](#terraform-provider)._
    ```hcl
    # example.tfvars
    address_group = "consul-services"
    tags = [
      "consul-terraform-sync",
      "terraform"
    ]
    ```
```

![image](https://user-images.githubusercontent.com/6362111/131385316-34e0096d-bb2f-44d2-b873-8fe7f6c94330.png)